xtext-website/documentation/310_eclipse_support.md

```diff
@@ -482,6 +482,61 @@ public Class<? extends IComparator>
 
 Xtext also provides a quick outline: If you press CTRL-O in an Xtext editor, the outline of the model is shown in a popup window. The quick outline also supports drill-down search with wildcards. To enable the quick outline, you have to put the [QuickOutlineFragment2]({{site.src.xtext_core}}/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ui/outline/QuickOutlineFragment2.xtend) into your workflow.
 
+## Folding {#folding}
+
+Xtext calculates the editor folding regions based on the grammar out of the box. Althought it comes with good defaults, sometimes they do not satisfy the needs and have to be customized.
+
+Considering e.g the Xtext Statemachine example, the framework provides folding capabilities only for the `state` regions:
```

Comment:
only nachstellen

Reply:
I would leave `only` exactly where it is, otherwise it sounds a little bit weird to me.

Reply:
@kthoms @szarnekow what do you think

Reply:
You can completely remove `only` from that sentence.

Reply:
done